src/Classes.java

```diff
@@ -0,0 +1,87 @@
+import java.util.ArrayList;
+
+public class Classes {
+	static ArrayList<Entity> entities;
+	static ArrayList<Relationship> relationships;
+
+	//********************************************************//
+	// Constructors                                           //
+	//********************************************************//
+	public Classes() {
+		entities = new ArrayList<Entity>();
+		relationships = new ArrayList<Relationship>();
+	}
+	
+	//********************************************************//
+	// Class Functions                                        //
+	//********************************************************//
+	public boolean createClass(String name) {
+		for (Entity e : entities) {
+			if (e.name.equals(name)) {
+				return false;
+			}
+		}
+		return entities.add(new Entity(name));
+	}
+
+	public boolean renameClass(String target, String newname) {
+		int index = entities.indexOf(new Entity(target));
+		if (index >= 0) {
+			entities.get(index).name = newname;
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	public boolean deleteClass(String target) {
+		return entities.remove(new Entity(target));
+	}
```

Comment:
Shouldn't this also delete any relationships tied to the class as well?

Reply:
That's a great find, Ian and I didn't implement the relationship functions yet so whenever we add those in we can definitely add that in as well.